contracts/cw3-flex-multisig/src/contract.rs

```diff
@@ -89,9 +89,10 @@ pub fn execute_propose(
     // only members of the multisig can create a proposal
     let cfg = CONFIG.load(deps.storage)?;
 
+    // Only members of the multisig can create a proposal
```

Comment:
Perhaps adding a comment here or below in `BALLOTS.save` would be a good idea. Mentioning that non-voting members are special in that they "vote" when creating proposals, but cannot vote normally. That vote is also special in that it has zero weight, but counts as a valid vote nevertheless (by example for threshold passing).